src/adapter/src/notice.rs

```diff
@@ -114,6 +116,7 @@ impl AdapterNotice {
     pub fn detail(&self) -> Option<String> {
         match self {
             AdapterNotice::PlanNotice(notice) => notice.detail(),
+            AdapterNotice::QueryTimestamp { explanation } => Some(format!("\n{explanation}")),
```

Comment:
do you want the newline?

Reply:
yes, otherwise the alignment of the first line is wrong in psql because it's prepended by `details:` or something